editor/editor_vcs_interface.cpp

```diff
@@ -30,132 +30,371 @@
 
 #include "editor_vcs_interface.h"
 
+#include "editor_node.h"
+
+#define UNIMPLEMENTED() ERR_PRINT(TTR("Unimplemented virtual function in EditorVCSInterface based node: ") + __func__)
```

Comment:
We normally don't translate console errors, so this shouldn't use `TTR`.
For errors displayed in the editor (e.g. as you do below with `EditorNode::show_warning()`) you should use `TTR` indeed.